### DISCUSSION TITLE
Ways to calibrate.

### DISCUSSION THREAD

**Discussion by DunaSpice** [mckaywrigley/chatbot-ui#1881] [Ideas]
Please step forward if you would like to collaborate on this project. In what way would you like to give and what is in it for you?

**Comment by smithinoz** [comment]
Thanks to Mckay, this was a great project that kick started many of our AI journeys.
I still use it for sentimental reasons, and tinker with the code, but seriously it just can't compete with the larger open source projects with hundreds if not thousands of active developers, adding dozens of new features regularly.  Most notable are artifacts and improvements to the rag pipelines. 
Don't get me wrong, this project still has many advantages such has an extremely low foot print and I will likely continue to tinker.

Also when I started, I needed to use custom models, but now the generic ones more than exceed my needs and capability.
Like how good is the just updated Claude Sonnet 3.5.

Personally have found it far more cost effective now to use someone else's service.  I was paying for Matt's back when usage was included, but he kept running out of credits.  So now I use another (I deliberately won't mention the name) but it has unlimited usage of all the main models (including music, videos and images) for around $6 a month.

I few calls to the new o1 series of models, I could easily spend that personally per day.  Guess it's a swings and roundabouts thing.

I stopped pushing to this repo, as they were never approved.  I guess if someone maintained a similar codebase I would likely contribute.
I am currently hosting on Docker and the standard supabase.co database.  I simplified the UI, made small changes like allowing the users to set the default model from the model select drop down.  But certainly nothing advanced.
And it's been super reliable, a few security issues but very small footprint.
If I was to allow users on, would need to setup some housekeeping.

**Reply by DunaSpice** [reply]
Thanks for reply I would also like to take this project further. I can sent up a repo or you. Maybe you can also help me figure out why I have errors if I host the project through NGINX. On the local, it works and on the remote it fails of logging in. Any clue?

**Comment by ndroo** [comment]
I'm running this app in a production environment as a much lesser feature of a larger platform.

A few things stand out to me in your comments @smithinoz
1. You reference larger projects? Any favourites you can mention? Given the dormant state of this one i might move over.
2. I am tempted to continue running with this one and just build on it. We have a private fork today but I would be more than happy to opensource it if others wanted to join in. We're about to start adding new features so its a decision moment for us as to if we adopt this code and run with it, or let it die and switch to something else.

**Comment by spammenotinoz** [comment]
Given this project is dormant I will mention two other projects;

Both support SSO and have active development (Web search, Claude style artefacts)

https://github.com/open-webui/open-webui/ (Licensing model is somewhat open source but with branding restrictions after the license change)
https://github.com/danny-avila/LibreChat (Was always the poor cousin, but is growing massively since the open-webui licensing changes)

Libre supports all the main providers out of the box, Open-WebUI requires middleware.
LibreChat is more of a ChatGPT experience, where Open-WebUI can scale to thousands of users, but every release has bugs so try and find a stable version (they are not branded as stable).

Open-WebUI you can fork prior to the license agreement change if branding is a concern, otherwise you can fork and modify but all Open-WebUI branding\logo placement\icons must not be altered.  The agreement spells this out.

No doubt we may see some good hard forks of Open-WebUI before the licensing change.

**Comment by Ability08035** [comment]
.com
